ci-operator/config/openshift/installer/master.json

```diff
@@ -12,6 +12,12 @@
       "namespace": "openshift",
       "name": "origin-v4.0",
       "tag": "base"
+    },
+    "base-smoke": {
+      "cluster": "https://api.ci.openshift.org",
+      "namespace": "openshift",
+      "name": "installer-origin-release",
+      "tag": "bazel"
```

Comment:
@clayton, I'm setting this up as an IS created from `cluster/ci/config/installer-origin-release-bazel.yaml`  then below I'm referencing a Dockerfile.ci and a from: base-smoke  to: installer-smoke, below

Reply:
If you're going to create a new release image for the long-term, please keep the naming consistent with other release images by storing this as a tag of openshift/release (create a new scheduled import of the bazel image as an image stream tag "bazel" on the "release" image stream).

All of the release images are create from build configs stored in this repo in projects/origin-release - you should probably create a build config definition there if this is going to live for for more than a few weeks.  If it isn't, then I can live with this once you fix the name -

Reply:
not sure the long-term plan of smoke tests and/or bazel..  I'll consult with the team - I suspect this is not a long term required image, but more a bridge in the process of moving bits from tectonic-installer to openshift-installer